test/extended/util/proxy.go

```diff
@@ -55,7 +55,7 @@ func RunProxy() {
 	proxy := goproxy.NewProxyHttpServer()
 	proxy.OnRequest(goproxy.ReqHostMatches(regexp.MustCompile("^.*$"))).HandleConnect(goproxy.AlwaysMitm)
 
-	logfile := filepath.Join("test", "integration", "out", "goproxylogfile.log")
+	logfile := filepath.Join("out", "goproxylogfile.log")
```

Comment:
:+1:

Reply:
cherry-picked this into the bundle update PR #3767